templates/run-once

```diff
@@ -12,6 +12,18 @@ cd ~
 # Install prerequsite packages
 env ASSUME_ALWAYS_YES=YES pkg install ansible git sudo
 
+# vagrant build will not use ansible pull on the jails (as it is better to just update on demand each jail)
+{% if is_vagrant %}
+# bootstrap access for vagrant user
+if [ ! -d /home/vagrant ]
+then
+pw useradd -n vagrant -s /bin/csh -m -G wheel
```

Comment:
Does it have to be csh? Can sh be used instead? Ansible doesn't work as well with non-POSIX shells.